lib/server/index.js

```diff
@@ -18,11 +19,15 @@ module.exports = ({ config = {}, port = null, filters = {} }) => {
   // before the `app.all('/broker/:id/*'` route
   app.get('/broker/:id', (req, res) => {
     const id = req.params.id;
+    logger.info({
+      method: req.method,
+      url: '/broker/:id/*',
+    }, 'request audit log');
```

Comment:
Maybe let go of this one? Since all the data is in the websocket, and the token is not logged, we don't really get much from this line.